### ISSUE TITLE
TemplateDoesNotExist in rest_framework with Django 1.10.1

### ISSUE THREAD

**Issue by FinnStutzenstein** [OpenSlides/OpenSlides#2375]
With the current master and current Django version (1.10.1) the rest framework could not be shown when navigating to `your.server/rest`. The framework in OpenSlides works. But for development, this is a bit annoying.
A temporary fix would be to stick to version 1.9.9 and wait until this is fixed in Django.

The Traceback:
```
Template loader postmortem
Django tried loading these templates, in this order:
Using engine :
    This engine did not provide a list of tried templates.

Traceback:

File "/home/finn/OpenSlides/.virtualenv/lib/python3.4/site-packages/django/core/handlers/exception.py" in inner
  39.             response = get_response(request)
File "/home/finn/OpenSlides/.virtualenv/lib/python3.4/site-packages/django/core/handlers/base.py" in _legacy_get_response
  249.             response = self._get_response(request)
File "/home/finn/OpenSlides/.virtualenv/lib/python3.4/site-packages/django/core/handlers/base.py" in _get_response
  217.                 response = self.process_exception_by_middleware(e, request)
File "/home/finn/OpenSlides/.virtualenv/lib/python3.4/site-packages/channels/handler.py" in process_exception_by_middleware
  227.             return super(AsgiHandler, self).process_exception_by_middleware(exception, request)
File "/home/finn/OpenSlides/.virtualenv/lib/python3.4/site-packages/django/core/handlers/base.py" in _get_response
  215.                 response = response.render()
File "/home/finn/OpenSlides/.virtualenv/lib/python3.4/site-packages/django/template/response.py" in render
  109.             self.content = self.rendered_content
File "/home/finn/OpenSlides/.virtualenv/lib/python3.4/site-packages/rest_framework/response.py" in rendered_content
  72.         ret = renderer.render(self.data, accepted_media_type, context)
File "/home/finn/OpenSlides/.virtualenv/lib/python3.4/site-packages/rest_framework/renderers.py" in render
  696.         template = loader.get_template(self.template)
File "/home/finn/OpenSlides/.virtualenv/lib/python3.4/site-packages/django/template/loader.py" in get_template
  25.     raise TemplateDoesNotExist(template_name, chain=chain)
Exception Type: TemplateDoesNotExist at /rest/
Exception Value: rest_framework/api.html
```

**Comment by normanjaeckel** [comment]
Temporary workaround: `pip uninstall Django && pip install "Django<1.10"`

**Comment by emanuelschuetze** [comment]
Fixed with #2373
